client/question/services/code_evaluators/PythonPrereqCheckService.js

```diff
@@ -89,32 +89,45 @@ tie.factory('PythonPrereqCheckService', [
     var detectAndGetWrongLanguageType = function(code) {
       var obscuredCode = getObscuredCode(code);
       var rawCodeArray = code.split('\n');
+      var lowestErrorCharNumber;
```

Comment:
Initialize appropriately to a very large number.

Reply:
In my draft I did initialized it, but then reworked it so I didn't have some arbitrary number set. I readded my high-number point. Done.